src/PKSim.UI/Views/Parameters/ParameterSetView.cs

```diff
@@ -89,7 +89,6 @@ public void Initialize(UxGridView gridView)
          InitializeWithGrid(_gridView);
          _gridViewBinder = new GridViewBinder<ParameterDTO>(_gridView)
          {
-            ValidationMode = ValidationMode.LeavingRow,
```

Comment:
That's why it was not working satisfactorily before. Without this, the validation shows up right when you try to add a halogen that makes effective weight below 0